src/pybind/mgr/nfs/module.py

```diff
@@ -130,3 +130,37 @@ def _cmd_nfs_cluster_config_set(self, cluster_id: str, inbuf: str) -> Tuple[int,
     def _cmd_nfs_cluster_config_reset(self, cluster_id: str) -> Tuple[int, str, str]:
         """Reset NFS-Ganesha Config to default"""
         return self.nfs.reset_nfs_cluster_config(cluster_id=cluster_id)
+
+    def fetch_nfs_export_obj(self):
+        return self.export_mgr
+
+    def fetch_cluster_obj(self):
+        return self.nfs
+
+    def is_active(self) -> bool:
+        return True
+
+    def export_ls(self) -> List[Dict[Any, Any]]:
+        return self.export_mgr.list_all_exports()
+
+    def export_get(self, cluster_id: str, export_id: int) -> Dict[Any, Any]:
+        return self.export_mgr.get_export_by_id(cluster_id, export_id)
+
+    def export_rm(self, cluster_id: str, pseudo: str) -> None:
+        self.export_mgr.delete_export(cluster_id=cluster_id, pseudo_path=pseudo)
+
+    def daemon_ls(self) -> List[Dict[Any, Any]]:
+        return self.nfs.list_daemons()
+
+    def cluster_ls(self) -> List[str]:
+        return [
+            {
+                'pool': NFS_POOL_NAME,
+                'namespace': cluster_id,
+                'type': 'orchestrator',
+                'daemon_conf': None,
+            } for cluster_id in available_clusters()
+        ]
```

Comment:
I have the feeling that the static checker is not working with the nfs module, since the return type of that should be something like `List[Dict[str, Optional[str]]]`.

Reply:
This method should be removed. Again importing nfs module is breaking cli. I have added patch along with comments.